website/docs/reference/operators/query/logical-operators/not.mdx

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 3
+description: Returns all documents that do not match the given query expression.
+---
+
+# `$not`
+
+import CodeBlock from '@theme/CodeBlock';
+import Dataset from '!!raw-loader!./logical-operators/2-insert-dataset.request.js';
+import NotOperatorRequest from '!!raw-loader!./logical-operators/5-not-operator.request.js';
+import NotOperatorResponse from '!!raw-loader!./logical-operators/5-not-operator.response.js';
+
+The `$not` operator negates a query expression and returns documents that do not match it.
+
+The operator accepts a single query expression as shown below, and all documents that do not match the expression are included in the result set.
+
+```js
+db.runCommand({
+  find: '<collection_name>',
+  filter: {
+    $not: { <query-expression> }
+  }
+})
+```
+
+A query expression can be any valid query such as `{ <field>: <value> }`, `{ <field>: { <operator>: <value> } }`, `{ <field>: /<pattern>/ }`, etc.
+
+## Example
+
+The following examples demonstrate how to use the `$not` operator in queries using documents in the following `books` collection:
+
+<!-- markdownlint-disable MD033 -->
+
+<CodeBlock language="js">{Dataset}</CodeBlock>
+
+### Filter documents that do not match a query expression
+
+Using the books collection, return all documents where the rating is not less than `4.5`.
```

Comment:
Could we use a bit simpler example? _not less than_ requires a bit of thinking, as we would prefer to use _greater than or equal_. Maybe something like not British?

Reply:
Thanks for asking this. I was able to go deeper and `$not` op only works for operator expressions and regex

Reply:
Can we use not equal then such as `$not: {$eq: xxxx}`? the usage of `not less than 4.5` is just weird as we would use `greater than or equal to 4.5`.